app/views/shared/_nav.html.erb

```diff
@@ -11,21 +11,38 @@
       <li class='nav-item'>
         <%= link_to 'Activity', activity_url, class: 'nav-link', data: { turbolinks: false } %>
       </li>
-      <% if can? :manage, User %>
-        <li class='nav-item'>
-          <%= link_to 'Users', site_users_url, class: 'nav-link' %>
-        </li>
-      <% end %>
     </ul>
     </div>
 
     <ul class="navbar-nav ml-auto">
       <% if current_user %>
-        <li class="nav-item">
-          <%= link_to current_user, edit_user_registration_path, class: 'nav-link' %>
-        </li>
-        <li class="nav-item">
-          <%= link_to 'Logout', destroy_user_session_path, class: 'nav-link', data: { method: :delete } %>
+        <li class="nav-item dropdown">
+          <a class="nav-link dropdown-toggle" href="#" id="navbarDropdown" role="button" data-bs-toggle="dropdown" aria-expanded="false">
+            <%= current_user %>
+          </a>
+          <ul class="dropdown-menu dropdown-menu-dark" aria-labelledby="navbarDropdown">
+            <% current_user.roles.each do |role| %>
+              <% if role.resource_type == "Organization" %>
+                <li class="nav-item">
+                  <% @organization = Organization.find(role.resource_id) %>
+                  <%= link_to "#{@organization.name} POD", @organization, class: 'nav-link' %>
+                </li>
+              <% end %>
```

Comment:
Maybe something like?:

```
current_user.roles.where(resource_type: 'Organization').each do |role|
  <li class="nav-item">
    <%= link_to "#{role.resource.name} POD", role.resource, class: 'nav-link' %>
  </li>
<% end %>
```

As an aside, when creating variables in templates it's usually a good idea to use local variables instead of instance variables (`organization` vs. `@organization`) to keep the scope limited.

Reply:
You can also use [rolify](https://github.com/RolifyCommunity/rolify#6-resource-roles-querying) to get the organizations, e.g.:

```ruby
Organization.with_roles([:owner, :member], current_user)
```

(although logic like that is probably a good candidate for pulling up to the `User` or `Organization` model?)